server/fleet/hosts.go

```diff
@@ -1109,7 +1109,7 @@ func ExpandPlatform(platform string) []string {
 const (
 	DeviceMappingGoogleChromeProfiles = "google_chrome_profiles"
 	DeviceMappingMDMIdpAccounts       = "mdm_idp_accounts"
-	DeviceMappingIDP                  = "idp"              // set by user via PUT /hosts/{id}/device_mapping with source=idp
+	DeviceMappingIDP                  = "idp"              // accepted as API input via PUT /hosts/{id}/device_mapping, stored as mdm_idp_accounts
```

Comment:
It should not be defined here (to reduce confusion). I would define it elsewhere with some history that it will be there because API compatibility, etc.